README.md

```diff
@@ -22,6 +22,38 @@ Alternatively, you can clone the package from GitHub and set everything up with
 Please note this package requires Tensorflow for Deepbooru tagging and other required packages which do take up quite a lot of disk space (>400MB).
 A future release of this package will offer an option without the need to install Tensorflow if you don't want to use Deepbooru tagging.
 
+### Docker Instructions
+If you would like to run the toolkit in a Docker container instead, follow the
+instructions below.
+
+1. Copy `docker-compose.yml` to the location where you want to run the toolkit.
+1. Copy `config_sample.toml` to the same location, renaming to `config.toml` and
+replacing with your configuration.
+1. Copy `crontab_sample` to the same location, renaming to `crontab` and adding
+   the commands you would like to run regularly. An example command is provided
+   in `crontab_sample`.
+1. Make sure to set the `src_path` option in `config.toml` to use
+   `/szurubooru-toolkit/upload_src`.
```

Comment:
I'd add that the user needs to modify the `/upload_src` in `docker-compose.yml`. This would also be the only time the user has to interact with that file.

Reply:
I also changed the temp directory in docker-compose.yml to be upload_src since it shouldn't be needed for anything, right?

Reply:
Yeah that's fine. I think I kept it as a reference in some places just in case, but I'll remove these soon as anything temp related is being executed in RAM.

Reply:
Would you be so kind and add the note for the `docker-compose.yaml` file I mentioned? I can then merge this PR 😅 .

Reply:
Hi, what note do you need? Docker should automatically create a folder at that location, so the only change needed should be setting that directory in `config.toml`, is there anything else needed?

Reply:
The current `docker-compose.yml` specifies on line 12  `- ./upload_src:/szurubooru-toolkit/upload_src`. The user needs to set `/szurubooru-toolkit/upload_src` in `config.toml`, yes - but he might also have to change `./upload_src` in case he has another dedicated directory for that (well, at least I do 😉 ).

So, the note should just tell the user that he might need to change the `./upload_src` in the `docker-compose.yml` file.

Reply:
Sorry for the delay, but I've added the note now, please let me know if there are any other changes you'd like!